Source/Fuse.Reactive.Bindings/Instance.API.uno

```diff
@@ -296,11 +296,28 @@ namespace Fuse.Reactive
 		
 		internal bool HasLimit { get { return _hasLimit; } }
 
-		protected internal object _items;
+		object _items;
+		/** @hide */
+		protected object GetItems() { return _items; }
+		/** @hide */
+		protected void SetItems( object value )
+		{
+			_items = value;
+			if (!IsRootingCompleted) return;
+			RefreshItems();
+		}
+		/** Call to set the items during the OnRooted override (post base.OnRooted call)
+			@hide */
+		protected void SetItemsDerivedRooting( object value )
```

Comment:
Typo, should be `SetItemsDuringRooting`, no?

Reply:
I actually used that before, but then decided the empahsis here is more on the "Derived" part, since other things can actually change the items during rooting, but only the derived class should call this.